### PR TITLE
Add Voice installers to ready-made projects

### PR DESCRIPTION
**Pull request by balloob**
## Description:
Add voice assistant to ready-made projects.

**Related issue (if applicable):** fixes <link to issue>

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#<esphome PR number goes here>

## Checklist:

  - [ ] I am merging into `next` because this is new documentation that has a matching pull-request in [esphome](https://github.com/esphome/esphome) as linked above.  
    or
  - [x] I am merging into `current` because this is a fix, change and/or adjustment in the current documentation and is not for a new component or feature.

  - [ ] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

File: projects/installer.html [hunk 1]
```diff
@@ -109,7 +112,14 @@
...
-    <input type="radio" name="type" value="bluetooth" checked />
+    <input type="radio" name="type" value="voice" checked />
+    <div class="name">Voice Assistant</div>
+    <div class="description">
+      Create a device to control Home Assistant with your voice.
+    </div>
+  </label>
+  <label>
+    <input type="radio" name="type" value="bluetooth" />
     <div class="name">Bluetooth Proxy</div>
```
Comment (c0ffeeca7) [inline projects/installer.html]:
```suggestion
    <div class="name">Bluetooth proxy</div>
```

File: projects/installer.html [hunk 2]
```diff
@@ -109,7 +112,14 @@
 <div class="question-prompt">I want to create a:</div>
 <div class="types">
   <label>
-    <input type="radio" name="type" value="bluetooth" checked />
+    <input type="radio" name="type" value="voice" checked />
+    <div class="name">Voice Assistant</div>
```
Comment (c0ffeeca7) [inline projects/installer.html]:
```suggestion
    <div class="name">Voice assistant</div>
```

File: projects/installer.html [hunk 3]
```diff
@@ -133,6 +143,248 @@
...
+        />
+        <div class="name">ESP32-S3-BOX-Lite</div>
+      </label>
+    </div>
+  </div>
+
+  <div class="question-prompt">Start the installation:</div>
+  <esp-web-install-button></esp-web-install-button>
+
+  <div class="question-prompt">Install Wake Word: (optional)</div>
```
Comment (c0ffeeca7) [inline projects/installer.html]:
```suggestion
  <div class="question-prompt">Install wake word: (optional)</div>
```

File: projects/installer.html [hunk 4]
```diff
@@ -133,6 +143,248 @@
...
+          href="https://www.aliexpress.us/item/3256805733893224.html?gatewayAdapt=glo2usa4itemAdapt"
+          >AliExpress</a
+        >
+      </li>
+    </ul>
+  </div>
+
+  <div class="hidden info quinled-dig2go">
+    <h3>dig2go</h3>
+    <p>Mini usb-c addressable LED controller.</p>
```
Comment (c0ffeeca7) [inline projects/installer.html]:
```suggestion
    <p>Mini USB-C-addressable LED controller.</p>
```

**Review by c0ffeeca7 — COMMENTED** [review]
Tiny tweaks

File: projects/installer.html [hunk 5]
```diff
@@ -133,6 +143,248 @@
...
+        />
+        <div class="name">ESP32-S3-BOX-Lite</div>
+      </label>
+    </div>
+  </div>
+
+  <div class="question-prompt">Start the installation:</div>
+  <esp-web-install-button></esp-web-install-button>
+
+  <div class="question-prompt">Install wake word: (optional)</div>
```
Comment (c0ffeeca7) [inline projects/installer.html]:
```suggestion
  <div><a
    href="https://www.home-assistant.io/voice_control/thirteen-usd-voice-remote/"
    target="_blank"
    >Set up a voice assistant and create your own wake word</a></div>
```

File: projects/installer.html [hunk 6]
```diff
@@ -133,6 +143,248 @@
...
+  <p>Install the add-on and configure it using the Wyoming integration.</p>
+  <a
+    href="https://my.home-assistant.io/redirect/supervisor_addon/?addon=core_openwakeword"
+    target="_blank"
+    rel="noreferrer noopener"
+    class="img"
+    ><img
+      src="https://my.home-assistant.io/badges/supervisor_addon.svg"
+      alt="Open your Home Assistant instance and show the dashboard of the Open Wake Word add-on."
+  /></a>
```
Comment (c0ffeeca7) [inline projects/installer.html]:
```suggestion
```

File: projects/installer.html (hunk 5 above)
Comment (balloob) [reply projects/installer.html]: The question prompt is the heading of steps. It feels weird to use it to link out to another page.

Comment (balloob) [reply projects/installer.html]: We also shouldn't use the word "Voice Remote" as it's a Voice Assistant.

Comment (balloob) [reply projects/installer.html]: I also don't think it's optional anymore, users need to go there to see how it is used ?

Comment (c0ffeeca7) [reply projects/installer.html]: yes to all your replies :smile:

**Pull request merged**